README.md

```diff
@@ -84,7 +82,8 @@ The current `package-storage` repository has a few branches. This is a quick sum
 * production: Packages for epr.elastic.co
 * staging: Packages for epr-staging.elastic.co
 * snapshot: Packages for epr-snapshot.elastic.co
-* experimental: Packages for epr-experimental.elastic.co. These packages are served by Kibana 7.8 and will disappear in the future. No updates should happen to this branch.
+* production-7.9: Packages for epr-7-9.elastic.co. These packages are served for Kibana 7.9 versions.
```

Comment:
I wonder if you should mention here that only bugfixes should be backported to this branch. The standard development for 7.10 should be performed in `production`.

Reply:
I added a more generic node that all future changes should go into production.